docs/content/en/2.components/1.nuxt-img.md

```diff
@@ -177,7 +176,7 @@ Some providers support other values
 
 ### `modifiers`
 
-In addition to standard modifiers, every provider can have their own modifiers.
+In addition to standard modifiers, every provider can have their own modifiers. This prop is only relevant when using [Image Services Providers](/providers#image-services-providers)
```

Comment:
This is not a right statement. ipx also has advanced modifiers not exposed as props

Reply:
I didn't know that, that's good news! Can I find any reference to IPX modifiers somewhere ?

Reply:
Not yet. PR welcome for docs. List is here: https://github.com/unjs/ipx/blob/main/src/handlers/handlers.ts

(list will be reduced if we use squoosh https://github.com/unjs/ipx/issues/31)